bin/kafka-server-stop.sh

```diff
@@ -19,6 +19,6 @@ if [ -z "$PIDS" ]; then
   echo "No kafka server to stop"
   exit 1
 else 
-  kill -SIGTERM $PIDS
+  kill -s TERM $PIDS
```

Comment:
@granders Make sure you're running a non-`bash` `/bin/sh`. I filed https://issues.apache.org/jira/browse/KAFKA-3384 not too long ago because this pops up in some cases -- the `-<SIGNALNAME>` variant actually isn't POSIX compliant, it is a shell extension.